### PR TITLE
Feature/final touches

### PR DESCRIPTION
**Pull request by Gnwin**
## Leaderboard: final-touches

In this project, I worked added styling while keeping the structure intact to make the app presentable based on the project requirements from:

https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m3_final_touches.md

This branch is a showcases the next milestone in the project based on project requirements:
- Improved the look and feel of the application, added the styles of my choice.
- Preserved general layout of the wireframe(mandatory).
- Used plain CSS.

Kindly awaiting permission to merge into develop branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,11 @@ A JavaScript project for the Leaderboard list app, using webpack and ES6 feature
...
 ## Screenshot
 
-<img src="./src/assets/images/Screenshot 2022-04-13 at 1.16.53 PM.png">
+<img src="./src/assets/images/Screenshot 2022-04-15 at 4.14.54 PM.png">
+
+<!-- ## Live Preview
+
+- Live preview: [click here](https://gnwin.github.io/Leaderboard/) -->
 
 ## Built With
```
Comment (EricMbouwe) [inline README.md]: - [x] README file should contain all that's needed for running the app with the API(s) it consumes. Let's add it

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
### Hi @Gnwin,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight!
- Good PR ✔️
- Good commits ✔️

# Required Changes ♻️ ⛔️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Gnwin) [reply README.md]: github-pages not working properly. css is not rendering

Comment (Gnwin) [reply README.md]: css is now rendering properly. gh-pages properly implemented. thanks

**Review by nicupop729 — APPROVED** [review]
<h1>Status: Approved 🥇 ✔</h1>
Hi @Gnwin,

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

### Some highlights for you
* Correct gitflow practice ✔️ 
* Good commit messages ✔️ 
* Professional README ✔️ 
* In the project:
   * Clean codebase ✔️ 
   * Good understanding of JavaScript concepts✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gnwin** [comment]
Thanks @nicupop729

**Pull request merged**